docs/datagatherers/gke.md

```diff
@@ -1,68 +1,62 @@
 # GKE Data Gatherer
 
-The GKE *data gatherer* fetches information about a cluster 
-from the Google Kubernetes Engine API.
+The GKE *data gatherer* fetches information about a cluster from the Google
+Kubernetes Engine API.
 
 ## Data
 
 The output of the GKE data gatherer follows the format described in the
 [GKE API reference](https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1beta1/projects.locations.clusters#Cluster)
 and the [Go Docs](https://godoc.org/google.golang.org/api/container/v1#Cluster).
-These are useful to check when writing new rules.
 
-The gathered data looks like this:
+It's comparable to the output from:
 
-```json
-{
-  "Cluster": {...}
-}
+```bash
+gcloud container clusters describe main --format=json
```

Comment:
```suggestion
gcloud container clusters describe my-cluster --format=json
```

Reply:
fixed in 6068de10553fed71f81bb8b64a350c0913776478